components/v-mapbox-components/v-mapbox-selected-point-layer.vue

```diff
@@ -43,13 +43,15 @@ export default {
     geometry(newValue) {
       const selectedLayer = this.map.getSource(this.selectedLayer.id)
       selectedLayer.setData(newValue)
+      this.map.moveLayer(this.selectedLayer.id)
```

Comment:
Move layer to top?

Reply:
Yes, this is the layer indicating the selection. Which was now each time underneath the mapboxlayer, which made it hard to see which point was selected.